javascript-conditionals/main.js

```diff
@@ -0,0 +1,93 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+function isUnderFive(number) {
+  return number < 5;
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH < 7 && pH >= 0) {
+    return 'acid';
+  } else if (pH > 7 && pH <= 14) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
+}
+
+function introduceWarnerBro(name) {
+  switch (name) {
+    case 'yakko':
+      return 'We\'re the warner brothers!';
+    case 'wakko':
+      return 'We\'re the warner brothers!';
+    case 'dot':
+      return 'I\'m cute~';
+    default:
+      console.log('Goodnight everybody!');
+  }
+}
+
+function recommendMovie(genre) {
+  switch (genre) {
+    case 'action':
+      return 'The Matrix';
+    case 'comedy':
+      return 'Meet the Parents';
+    case 'horror':
+      return 'The Exorcist';
+    case 'drama':
+      return 'A Beautiful Mind';
+    case 'musical':
+      return 'The Sound of Music';
+    case 'sci-fi':
+      return 'Interstellar';
+    default:
+      console.log('Goodnight everybody!');
```

Comment:
this should return the string "Genre not recognized. Choose between action, comedy, horror, drama, musical, or sci-fi"